kalbarri-workshop.md

```diff
@@ -342,6 +344,7 @@ print(diff)
 
 You should see the result of calling `subtraction()` printed in the *Console*.
 
+<!--Stacking of bands will be unfamiliar to a newcomer at this point in the lab - consider a different function or shifting this to where geographic data starts to be used -->
```

Comment:
Not sure I'd have any context to what `addBands()` is if I had no prior remote sensing experience - maybe another function here that's more commonplace?